### ISSUE TITLE
Notebook không dùng được, có thể là do coursera update

### ISSUE THREAD

**Issue by GlorYght** [vinhthanai/NLP301c#1]
![image_2024-10-23_135702462](https://github.com/user-attachments/assets/d72988b2-dcef-410b-bc00-56f56debe690)
Cụ thể là C3W3, em sẽ ngó thử code xem khác chỗ nào sau nhưng em nghĩ vẫn nên báo cho anh một câu.
Btw, thanks vì source code

**Comment by GlorYght** [comment]
ở hàm # GRADED FUNCTION: train_model em thấy bản mới nhất bên coursera có thay train_steps thành epochs
sau khi thay tên biến rồi chạy lại thì pass ngon, cho dù kết quả notebook vẫn thế.
![image_2024-10-23_143148792](https://github.com/user-attachments/assets/0c5635d6-3f8a-44e2-b13d-26acdaca35f7)

**Comment by GlorYght** [comment]
`# GRADED FUNCTION: train_model
def train_model(Siamese, TripletLoss, text_vectorizer, train_dataset, val_dataset, d_feature=128, lr=0.01, epochs=5):
    """Training the Siamese Model

    Args:
        Siamese (function): Function that returns the Siamese model.
        TripletLoss (function): Function that defines the TripletLoss loss function.
        text_vectorizer: trained instance of `TextVecotrization` 
        train_dataset (tf.data.Dataset): Training dataset
        val_dataset (tf.data.Dataset): Validation dataset
        d_feature (int, optional) = size of the encoding. Defaults to 128.
        lr (float, optional): learning rate for optimizer. Defaults to 0.01
        epochs (int): number of epochs
        
    Returns:
        tf.keras.Model
    """
    ## START CODE HERE ###

    # Instantiate your Siamese model
    model = Siamese(text_vectorizer,
                    vocab_size = text_vectorizer.vocabulary_size(), #set vocab_size accordingly to the size of your vocabulary
                    d_feature = d_feature)
    # Compile the model
    model.compile(loss=TripletLoss,
                  optimizer = tf.keras.optimizers.Adam(learning_rate=lr)
            )
    # Train the model 
    model.fit(train_dataset,
              epochs = epochs,
              validation_data = val_dataset,
             )
             
    ### END CODE HERE ###

    return model`

**Comment by vinhthanai** [comment]
Cảm ơn em đã report, anh đã check lại và update file notebook
